src/dal/Jalasoft.TeamUp.Resumes.DAL/ResumesRepository.cs

```diff
@@ -4,6 +4,7 @@
     using System.Collections.Generic;
```

Comment:
this repository should not be modified

Reply:
Since IRepositoty was updated; this class was update too. I have to implement GetAll, GetById y Add to keep the funtionality.